docs/pipeline.rst

```diff
@@ -95,8 +109,8 @@ commit of all of the external Git repos in use by the site and how long ago they
 version of the database in use, the git revision of the bedrock code, and how long ago the database was updated. If you recently made
 a change to one of these repos and are curious if the changes have made it to production, this is the URL you should check.
 
-Updating Selenium
-~~~~~~~~~~~~~~~~~
+Updating Selenium (todo)
```

Comment:
@glogiotatidis I'm guessing this section is out of date. Do we need to change anything here on how to update the selenium docker images?

Reply:
The current local grid setup always uses the latest selenium version from the 3.x series. I understand from [selenium docs](https://wiki.saucelabs.com/display/DOCS/_selenium_version), that it will chose the best version based on the OS/Browser combination. We don't need to update something else at this point. Does this answer your question @alexgibson ?